src/Emitter/SapiEmitter.php

```diff
@@ -10,6 +10,13 @@
 final class SapiEmitter implements EmitterInterface
 {
     private const NO_BODY_RESPONSE_CODES = [204, 205, 304];
+    private $bufferSize;
+    private const DEFAULT_BUFFER_SIZE = 8388608; // 8MB
+
+    public function __construct(int $bufferSize = self::DEFAULT_BUFFER_SIZE)
+    {
+        $this->bufferSize = $bufferSize > 0 ? $bufferSize : self::DEFAULT_BUFFER_SIZE;
```

Comment:
If zero or negative, the default approach is not really good. I suggests
- `?int .. = null` for parameter
- if negative or zero, throw InvalidParameterException
- if null then default.